database-docker/docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
     environment:
       POSTGRES_USER: postgres
       POSTGRES_PASSWORD: postgres
-      POSTGRES_DB: dora
+      POSTGRES_DB: mhq-oss
```

Comment:
```suggestion
      POSTGRES_DB: middleware
```

Reply:
The whole application uses `mhq-oss` in all environments. This shall mean change over their as well